Tweeter/Content/Site.css

```diff
@@ -1,24 +1,79 @@
-﻿body {
-    padding-top: 50px;
-    padding-bottom: 20px;
-}
-
-/* Set padding to keep content from hitting the edges */
-.body-content {
-    padding-left: 15px;
-    padding-right: 15px;
-}
-
-/* Override the default bootstrap behavior where horizontal description lists 
-   will truncate terms that are too long to fit in the left column 
-*/
-.dl-horizontal dt {
-    white-space: normal;
-}
-
-/* Set width on the form input elements since they're 100% wide by default */
-input,
-select,
-textarea {
-    max-width: 280px;
-}
+﻿body {
+    padding-top: 60px;
+    padding-bottom: 20px;
+}
+
+/* Set padding to keep content from hitting the edges */
+.body-content {
+    padding-left: 15px;
+    padding-right: 15px;
+}
+
+/* Override the default bootstrap behavior where horizontal description lists 
+   will truncate terms that are too long to fit in the left column 
+*/
+.dl-horizontal dt {
+    white-space: normal;
+}
+
+/* Set width on the form input elements since they're 100% wide by default */
+input,
+select,
+textarea {
+    max-width: 280px;
+}
+
+.tweets__container {
+    margin-top: 20px;
+}
+
+.tweets__tweet--image {
+    padding: 20px;
+    border: 2px solid #337AB7;
+    border-radius: 5px;
+}
+
+.tweet__image {
+    max-width: 100%;
+    border-radius: 5px;
+}
+
+.nav__input-container {
+    max-width : 300px;
+}
+
+.placeholder-tweet-message-container-class {
+    margin-top: 15px;
+    padding-top: 5px;
+}
+
+hr {
+    border-color: #337AB7;
+}
+
+.user__profile__card--upper {

```

Comment:
Way to use **BEM** naming. Here's a good example from Lonely Planet:

http://rizzo.lonelyplanet.com/styleguide/css-utilities/proportional-grid